aidb/inference/bound_inference_service.py

```diff
@@ -64,21 +64,21 @@ def get_table(conn: sqlalchemy.engine.base.Connection):
       fk_constraints = {}
       for column_name in self.binding.input_columns:
         column = self._columns[column_name]
-        new_table_col_name = self.convert_normalized_col_name_to_cache_col_name(column_name)
-        logger.debug('New table col name', new_table_col_name)
-        logger.debug('Ref column', str(column))
-        fk_ref_table_name = str(column).split('.')[0]
-        if fk_ref_table_name not in fk_constraints:
-          fk_constraints[fk_ref_table_name] = {'cols': [], 'cols_refs': []}
-        # both tables will have same column name
-        fk_constraints[fk_ref_table_name]['cols'].append(new_table_col_name)
-        fk_constraints[fk_ref_table_name]['cols_refs'].append(column)
-        columns.append(sqlalchemy.schema.Column(new_table_col_name, column.type))
+        if column.primary_key:
```

Comment:
For the cache table, store only the primary key to avoid foreign key constraint errors.